### ISSUE TITLE
Feature request "bike mode"

### ISSUE THREAD

**Issue by DC4FRT** [richonguzman/LoRa_APRS_Tracker#74]
Hello Ricardo, I have a feature request. 
Could you include another display page "Bike Mode"? 
I attached the tracker to my bicycle handlebars and can not read so much during driving.
But i actually only need the call sign or SSID or symbol in small letters (so that i know i have the correct SSID chosen)
and then the current speed in large letters (filling up the display). This is then like a Bicycle computer.

thanks so much for your great work !
55 and 73 Thomas, DC4FRT

**Comment by richonguzman** [comment]
good idea Thomas, will see how to do this and maybe add this as a "special" screen chosen in every callsign
